docs/static/filebeat-modules.asciidoc

```diff
@@ -0,0 +1,321 @@
+[[filebeat-modules]]
+
+== Working with Filebeat Modules
+
+Starting with version 5.3, Filebeat comes packaged with pre-built 
+{filebeat}filebeat-modules.html[modules] that contain the configuration needed
+to read, parse, and visualize data from various log file formats, such as Nginx,
+Apache2, and MySQL. Each Filebeat module consists of one or more filesets that
+contain ingest node pipelines, Elasticsearch templates, Filebeat prospector
+configurations, and Kibana dashboards.
+
+Filebeat modules do not currently provide Logstash pipeline configurations.
+In the future, Filebeat modules will provide tighter integration with Logstash
+to offer you a more powerful alternative to using ingest node pipelines.
+For now, you can follow the steps in this section to configure Filebeat and
+build Logstash pipeline configurations that are equivalent to the ingest
+node pipelines available with the Filebeat modules.
+
+Then you'll be able to use the sample Kibana dashboards available with Filebeat
+to visualize your data in Kibana.
+
+NOTE: These manual steps will no longer be required when Logstash support
+is added to Filebeat modules in a future release.
+
+To build and run Logstash configurations that provide capabilities similar to
+Filebeat modules:
+
+. Load the Filebeat index pattern and sample Kibana dashboards.
++
+To do this, you need to run Filebeat with the Elasticsearch output enabled and
+specify the `-setup` flag. For example:
++
+[source,shell]
+----------------------------------------------------------------------
+./filebeat -e -setup -E "output.elasticsearch.hosts=["http://localhost:9200"]"
```

Comment:
I think @suyograo is right here, the `-modules=nginx` flag is needed and works because this is specifically meant to use the ES output (`-E "output.elasticsearch.hosts=["http://localhost:9200"]"`).